fv3core/fv3core/stencils/d_sw.py

```diff
@@ -1046,12 +1048,18 @@ def __call__(
             self._tmp_fy,
         )
 
+        # TODO: part of flux_capacitor_stencil (updating cx, cy)
+        # should be mergeable with fv_prep, the other part (updating xflux, yflux)
+        # should be mergeable with any compute domain stencil in this object
+
         self._flux_capacitor_stencil(
             cx, cy, mfx, mfy, crx, cry, self._tmp_fx, self._tmp_fy
         )
 
         if not self.hydrostatic:
 
+            # TODO: output value for tmp_wk here is never used, refactor so it is
+            # not unnecessarily computed
```

Comment:
I think the fortran code commonly uses `work`, `wk` or some work combination names to reuse storages. It's great that you're able to decouple this one here. We should probably go through all of them at some point to see if they can all be temporaries, fused into one stencil, or not even compute it (I think sometimes they are calculated to save as diagnostics).